ComponentKit/Core/CKComponentViewConfiguration.mm

```diff
@@ -20,7 +20,12 @@
 
 CKComponentViewClass::CKComponentViewClass(Class viewClass) noexcept :
 identifier(class_getName(viewClass)),
-factory(^{return [[viewClass alloc] init];}) {}
+factory(^{
+  #if DEBUG
+    CKAssertTrue([viewClass isSubclassOfClass:[UIView class]], @"%@ is not a subclass of UIView", viewClass);
+  #endif
+  return [viewClass isSubclassOfClass:[UIView class]] ? [[viewClass alloc] init] : nil;
+}) {}
```

Comment:
Is this enough though? When we are using component kit as a precompiled dependency won't it be compiled without `DEBUG`?